lms/djangoapps/instructor/views/gradebook_api.py

```diff
@@ -15,12 +17,33 @@
 from courseware.courses import get_course_with_access
 from instructor.offline_gradecalc import student_grades
 from instructor.views.api import require_level
-
+from instructor_task.models import ReportStore
 
 # Grade book: max students per page
 MAX_STUDENTS_PER_PAGE_GRADE_BOOK = 20
 
 
+def list_report_downloads_by_course_key(course_key):
+    """
+    Return json list of reports like grade book, ready for download
+
+    Arguments:
+        course_key (CourseKey): Course key
+
+    Returns:
+      string: Json list of reports ready for download.
+    """
+    report_store = ReportStore.from_config(config_name='GRADES_DOWNLOAD')
+
+    response_payload = {
+        'downloads': [
+            dict(name=name, url=url, link='<a href="{}">{}</a>'.format(url, name))
```

Comment:
I dont know, These links are consumed by [Slick library in javascript code](https://github.com/edx/edx-platform/blob/master/lms/static/coffee/src/instructor_dashboard/util.coffee#L356).

I just moved this code to common place so that 2 apps (CCX, instructor) can access it 
https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor/views/api.py#L2204